.github/workflows/pull.yml

```diff
@@ -263,27 +263,14 @@ jobs:
           { config: "default", shard: 2, num_shards: 2, runner: "windows.4xlarge" },
         ]}
 
+  # please ensure that this and its corresponding job in trunk.yml are the same
   win-vs2019-cuda11_3-py3-build:
```

Comment:
maybe add `if: github.event_name == 'pull_request'` + a comment so that we don't run the build twice on master. And